app/src/main/java/com/kylecorry/trail_sense/main/MainActivity.kt

```diff
@@ -108,6 +112,25 @@ class MainActivity : AndromedaActivity() {
         }
     }
 
+    fun changeBottomNavLabelVisibility(useCompactMode: Boolean){
+        cache.putBoolean("pref_use_compact_mode", useCompactMode)
```

Comment:
Can you use `userPrefs.useCompactMode = useCompactMode` instead? That way if I ever need to change the string resource, it is all in one place.

(the BooleanPreference supports read and write - you'll just need to change it from `val` to `var`)

Reply:
Yes, fixed.